frontend/src/js/controllers/challengeCreate.js

```diff
@@ -71,6 +72,10 @@
                         }
                     };
                 }
+                vm.isExistLoader = true;
```

Comment:
Please declare this variable above(where all the initial declarations are there) with value `false`.